MarkupsToModel/Logic/vtkSlicerMarkupsToModelLogic.cxx

```diff
@@ -367,10 +395,14 @@ void vtkSlicerMarkupsToModelLogic::UpdateOutputCloseSurfaceModel( vtkMRMLMarkups
     {
       // draw a "square" around the line (make it a rectangular prism)
       vtkSmartPointer<vtkRegularPolygonSource> squareSource = vtkSmartPointer<vtkRegularPolygonSource>::New();
-      double extrusionMagnitude = ComputeSurfaceExtrusionAmount( cleanedPoints ); // need to give some depth
       squareSource->SetCenter( 0.0, 0.0, 0.0 );
+      double extrusionMagnitude = ComputeSurfaceExtrusionAmount( smallestBoundingExtentRanges ); // need to give some depth
       squareSource->SetRadius( extrusionMagnitude );
       squareSource->SetNumberOfSides( 4 );
+      double lineAxis[ 3 ] = { 0.0, 0.0, 0.0 }; // temporary values
+      const int LINE_AXIS_INDEX = 0; // The largest (and only meaningful) axis is in the 0th column
+      // the bounding axes are stored in the columns of transformFromBoundingAxes
+      GetNthAxisInMatrix( transformFromBoundingAxes, LINE_AXIS_INDEX, lineAxis ); 
```

Comment:
GetNthAxisInMatrix is not clear. GetNthColumnOfMatrix would be better. But overall, this helper function just makes things longer. Instead of setting the normal in 4 lines with the help of another function, you could do it all in-place, in one line (it's a long line, so you can break it into 3, but it'll be still shorter and you don't need the additional function):

squareSource->SetNormal( transformFromBoundingAxes->GetElement(0, LINE_AXIS_INDEX), transformFromBoundingAxes->GetElement(1, LINE_AXIS_INDEX), transformFromBoundingAxes->GetElement(2, LINE_AXIS_INDEX) );